packages/angular/src/app/pages/planning-scheduler/planning-scheduler.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="view-wrapper">
+  <div class='panels'>
+    <left-side-panel class="left-side-panel">
+      <div class="left-side-panel-content">
+        <div class='buttons'>
+          <dx-button text='Today'
+                     (click)="onSelectedDateChange()"
+          ></dx-button>
+          <dx-button text='Create event'
+                     type='default'
+                     (click)="showAppointmentCreationForm()"
+          ></dx-button>
+        </div>
+        <div class='calendar'>
+          <dx-calendar [value]="currentDate" (valueChange)="onCalendarDateChange($event)" ></dx-calendar>
+        </div>
+        <calendar-list [dataSource]="listDataSource"
+                       (listSelectionChanged)="calendarListChanged($event)"></calendar-list>
+      </div>
+    </left-side-panel>
+
+    <div class='scheduler-container'>
+      <dx-scheduler #schedulerRef
+                    [adaptivityEnabled]="!!((screen.screenChanged | async)?.isXSmall)"
+                    [allDayPanelMode]="'hidden'"
+                    [dataSource]="tasks"
+                    [currentView]="currentView"
+                    [currentDate]="getSchedulerCurrentDate | apply: currentDate"
+                    (currentViewChange)="onCurrentViewChange($event)"
+                    height='inherit'
```

Comment:
looks unstructured. Could you please structure properties like: simple binding, one-way binding and then emits?